docusaurus/docs-internals/editor-controls/data-controls/select.md

```diff
@@ -0,0 +1,406 @@
+| `choices`          | `ChoicesSync \| ChoicesAsync`                                                                                                                                                 |    -    | The `choices` property can be configured to either synchronously or asynchronously provide a list of selectable options for a given component. This property can take two forms: `ChoicesSync` or `ChoicesAsync`.<br/><br/>**ChoicesSync**<br/><br/>`ChoicesSync` is an array of `Choice` objects that are directly provided to the component.<br/><br/>**`Choice: { icon?: { name?: string; className?: string; }; title: string; value: Literal; }`**<br/><br/> Each `Choice` object defines the following properties:<br/>- `icon` (optional): An object specifying the icon details for the choice.<br/>  - `name` (optional): The name of the icon.<br/>  - `className` (optional): The CSS class name for styling the icon.<br/>- `title`: A string representing the display title of the choice.<br/>- `value`: A literal value representing the choice. This value is returned when the choice is selected.<br/><br/>**ChoicesAsync**<br/><br/>`ChoicesAsync` is used to load the choices dynamically via an asynchronous function. It is an object that defines the following properties:<br/><br/>**`ChoiceAsync: { load: (abortSignal?: AbortSignal) => Promise<Choice[]>; emptyLoad?: { title?: string; }; }`**<br/><br/>- `load`: A function that returns a `Promise` resolving to an array of `Choice` objects. This function can optionally take an `AbortSignal` to handle aborting the loading process.<br/>- `emptyLoad` (optional): An object that provides default text to display when no choices are loaded.<br/>  - `title` (optional): A string representing the text to display when the choices list is empty. |
```

Comment:
instead of "(optional)" add "?", example:
title (optional) -> title?

apply this changes also for `emptyLoad` and other parameters where you use "(optional)"